### ISSUE TITLE
[Request] People Ship Contributions

### ISSUE THREAD

**Issue by Zitchas** [endless-sky/Endless-Sky-Delta#135]
### Problem Description

In-game we have currently a few classes of ships:
1) Normal stock ships
2) Variant ships
3) Mission ships
4) Person ships.

Stock are supposed to be the "from the dealership state." I personally think they tend to be a bit too good, too well optimized for this, but anyway. The default configuration.

Variant ships are used to create a variety of variations on the stock ships. Configurations that are very common, but aren't stock.

Mission ships are generally that, but more customized and focused for that mission.,

People ships are all over the map, ranging from "could be a regular ship people can buy like the Lampyrid to things like the Heron.

### Desired Solution

What I would like to see is that "people ships" move away from being exclusively rewards for people who contribute a lot to the game, and more used as a tool to make the in-game universe feel much more interesting, diverse, and alive. As such, I would like to use people ships to create a vast pool of "unique, but 100% player-replicable configurations" that would add an immense amount of depth and interest to the world. Not to mention make the outfit scanner much more valuable and interesting to use. Things that could actually inspire the player to build their ship like them, or show what is possible.

So, for all of you who would like to contribute something to the game, here's something that you can do: Make a person ship. For this project, I would like to get game-legal ships representative of early-to-mid game (or that could reasonably be considered such for other factions if you started the game in the Coalition, or Hai, or wherever space). We don't need hundreds of people flying heavy warships. We definitely need some, but not as many as we need medium sized ships. Mediums is probably the highest priority.

Steps:
1. Pick a faction that you'd like to belong to. (preference is for Human, Hai, Pirate, and Coalition, as those need the most depth and diversity, but others are acceptable.)
2. Pick a ship that commonly shows up in that faction's space.
3. Outfit the ship the way you would personally outfit, but restricted to what you can typically acquire in the regions that faction frequents. Human ships will be mostly human outfits, maybe with some Hai. Hai ships will be mostly Hai, but possibly some human too. Coalition will be exclusively coalition outfits, Remnant will be mostly Remnant but with some Korath or Ka'het things, etc.
4. Write up a dozen or so lines or phrases that represent you-as-your-pilot.
5. Put it into the person file as a person with a variant ship. (so not a new ship, it should be a variant of an existing ship)
6. Submit the PR to here!

If you need help, please use the discussion area to ask for help, or feel free to DM me. Github can be kind of intimidating or obscure the first few times using it; but this is a great no-pressure sort of contribution to get started with. I, and many others, are happy to help.

(I personally work on Windows and primarily use Github Desktop to make life much easier, but others work on Linux or Mac. Whatever you use, chances are someone can help  you.)

# Notes

- These person ships will be restricted to the areas of space that they could normally reach, and will appear to the player like a uniquely configured but otherwise normal ship for the chosen faction.
- These ships should not be spoilers or have spoiler outfits.

**Comment by TheGiraffe3** [comment]
We'd also want to change the person spawn rate up, correct?
Otherwise you'll see the same ship once out of every thirty IRL hours you play. While I agree more variety is good, I think it'd be nice to see familiar people somewhat frequently.

And a question: are you only allowed to create a person for yourself? Similar to #134, I think I have a few ideas (both for human and non-human) ships that wouldn't be something I might use myself, but would be fun to have and easy to make quotes for.
Basically persons that don't represent anybody?

**Comment by Zitchas** [comment]
As an example of adding a person ship. (albeit not a perfect one, since this one does actually make a new variant with massively extra guns and stuff. Which isn't quite what I want for general person ships.) https://github.com/endless-sky/Endless-Sky-Delta/pull/134

Rephrasing that:
- You can create ships for yourself
- You can create ships for fictional people.
- You can**not** create ships for other real people without them posting their consent on here.

And yes, you do have a good point. As more of these "legal" person ships are added, we will shorten the timer on the person spawn rate so they show up more often.

As a general rule, no real world info that might contribute to Doxing please.
